tests/e2e/default/enketo/undo-death-report.wdio-spec.js

```diff
@@ -0,0 +1,64 @@
+const utils = require('../../../utils');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const userFactory = require('../../../factories/cht/users/users');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const personFactory = require('../../../factories/cht/contacts/person');
+const contactPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+const genericForm = require('../../../page-objects/default/enketo/generic-form.wdio.page');
+const reportsPage = require('../../../page-objects/default/reports/reports.wdio.page');
+const analyticsPage = require('../../../page-objects/default/analytics/analytics.wdio.page');
+const deathReportForm = require('../../../page-objects/default/enketo/death-report.page');
+const undoDeathReportForm = require('../../../page-objects/default/enketo/undo-death-report.page');
+
+describe('Submit an undo death report', () => {
+  const places = placeFactory.generateHierarchy();
+  const healthCenter = places.find(place => place.type === 'health_center');
```

Comment:
We're doing this more often. I think `placeFactory.generateHierarchy()` can return a map then we just access the object like this:
```
const places = placeFactory.generateHierarchy();
const healthCenter = places['health_center'];  // Regular js object
// Or 
const healthCenter = places.get('health_center'); // Actual Map type object.
```

Reply:
That is a really good idea. I implemented the changes, but I am not sure if I did it in the correct way, or if something should be different/better. Could you please check?
Thanks!

Reply:
Looks good, I took a quick look :)